docs/release/release_0_4_17.md

```diff
@@ -12,8 +12,40 @@ https://github.com/napari/napari
 
 ## Highlights
 
-- Feature: Minimum blending (#4875)
-- Add option to edit second shortcut in settings (#5018)
+This release is focused on documentation improvements and bug fixes, with few changes to napari’s API.
+We picked out a few highlights, but keep reading below for the full list of changes!
+
+### Minimum blending for a better light theme experience
+
+We added a new minimum blending option in #4875 that is particularly useful for those
```

Comment:
@psobolewskiPhD : I think you're busy at the moment, so I added a short blurb about this new feature. But feel to suggest corrections and additions!

Reply:
Thanks for the heads up and the opportunity ♥️ 
I expanded a bit to include mention of the inverted colormaps from @cleterrier 
I used suggestions, a bit awkwardly. Feel free and ignore of course or tweak as you see fit.

Reply:
Suggestions were perfect. I committed them then followed up with a little rewording.